apps/web/server/lib/ssr.ts

```diff
@@ -26,6 +26,8 @@ export async function ssrInit(context: GetServerSidePropsContext) {
 
   // always preload "viewer.public.i18n"
   await ssr.viewer.public.i18n.fetch();
+  // So feature flags are available on first render
```

Comment:
I chose this common helper instead of each individual page because prefetched feature flags would be desirable in most cases.